client/js/render.js

```diff
@@ -305,18 +315,24 @@ function loadMoreHistory(entries) {
 	});
 }
 
-sidebar.on("click", ".collapse-network", (e) => {
-	const collapseButton = $(e.target);
+sidebar.on("click", "button.collapse-network", (e) => collapseNetwork($(e.target)));
```

Comment:
I'd remove `button` from this selector as well.

Reply:
I feel better having it in because we can never be too specific with these things and it's not like it makes it less efficient, but could stop bugs happening in the future.